src/index.js

```diff
@@ -10,5 +11,6 @@ export const LegacyComponents = {
 export const StandardComponents = {
     Button,
     Image,
-    Text
+    Text,
+    ElementBlock
```

Comment:
This component will not be available through `ElementPropTypes.component()`, so it should not be included in the `StandardComponents` object. For now, we should export it from the top level of the module.

Reply:
f98d70e34dab56e1b6221b39cb5bf26014fd78fd